src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -2028,23 +2028,19 @@ UpdateShardPlacement(PlacementUpdateEvent *placementUpdateEvent,
 	if (updateType == PLACEMENT_UPDATE_MOVE)
 	{
 		appendStringInfo(placementUpdateCommand,
-						 "SELECT citus_move_shard_placement(%ld,%s,%u,%s,%u,%s)",
```

Comment:
You forgot to update the calls used by the background rebalancer (line 1954). Also while not strictly necessary it would be good to update CopyShardPlacementToWorkerNodeQuery